conf/messages.en

```diff
@@ -175,7 +175,7 @@ error.epu.required=Enter an EPU number
 error.epu.invalid-length=EPU number must be 3 characters
 error.epu.invalid-only-digits=EPU number must only contain numbers 0 to 9
 error.epu.invalid-number=EPU number must be 700 or less
-error.entryNumber.required=Enter an entry number
+error.entryNumber.required=Entry number
```

Comment:
Sorry but this will need to go back to how it was lol (this is the error message). 

There are changes needed on lines: 
176 - EPU number must only contain numbers 

180 - (similar but for Entry No)

190, 191 & 192 - similar change but we also want to make them a little more specific so each one includes day/month/year ie. 
190 (year) => Entry **year** must only contain numbers

Basically we're just dropping the bit that says 0-9 because it's a little redundant

Something similar will also be happening with the "Entry date must be a real date" however this will be done in another story so **don't** worry about it for this as the logic may need a tweak - just mentioning so you're aware


Finally - the spreadsheet has the following error for when a user puts in a date in the future: 
Entry date must be in the past 

Again this is something I wouldn't change right now because I think what it is currently is actually a little more informative (Entry date must be today or in the past) & I think we might want to add a bit to say 6months in the past - this I'm not sure on and tried to chat with Irene about today but I missed her - anyway - I'd leave it for now & I'll discuss this with her once she returns (not a big deal anyway & can be changed easily enough if we need to)

Reply:
I copied these over so they should be accurate BUT do cross check it with this https://docs.google.com/spreadsheets/d/16ix1ylUxW9mymYW5QWn-NhcnSgLH05pMkIw6bcTEcBQ/edit?pli=1#gid=1444330108